src/Simplex/Messaging/Agent/Store/SQLite.hs

```diff
@@ -452,85 +619,99 @@ insertSndQueue_ dbConn SndQueue {..} = do
     [ ":host" := host server,
       ":port" := port_,
       ":snd_id" := sndId,
-      ":conn_alias" := connAlias,
+      ":conn_alias" := connId,
       ":snd_private_key" := sndPrivateKey,
       ":encrypt_key" := encryptKey,
       ":sign_key" := signKey,
       ":status" := status
     ]
 
-insertSndConnection_ :: DB.Connection -> SndQueue -> IO ()
-insertSndConnection_ dbConn SndQueue {server, sndId, connAlias} = do
+insertSndConnection_ :: DB.Connection -> ConnData -> SndQueue -> IO ()
+insertSndConnection_ dbConn ConnData {connId, viaInv, connLevel} SndQueue {server, sndId} = do
   let port_ = serializePort_ $ port server
   DB.executeNamed
     dbConn
     [sql|
       INSERT INTO connections
-        ( conn_alias, rcv_host, rcv_port, rcv_id, snd_host, snd_port, snd_id,
+        ( conn_alias, rcv_host, rcv_port, rcv_id, snd_host, snd_port, snd_id, via_inv, conn_level,
           last_internal_msg_id, last_internal_rcv_msg_id, last_internal_snd_msg_id,
           last_external_snd_msg_id, last_rcv_msg_hash, last_snd_msg_hash)
       VALUES
-        (:conn_alias,     NULL,     NULL,   NULL,:snd_host,:snd_port,:snd_id,
+        (:conn_alias, NULL, NULL, NULL, :snd_host, :snd_port, :snd_id, :via_inv, :conn_level,
```

Comment:
These were easy to verify visually when they had the same horizontal indentation